.travis.yml

```diff
@@ -1,42 +1,72 @@
 language: c
 
-_linuxpackages: &_linuxpackages
-  addons:
-    apt:
-      packages:
-        - build-essential
-        - git
-        - autoconf
-        - automake
-        - autopoint
```

Comment:
These should remain, just in case: here on Debian sid amd64, neither build-essential, nor libtool, nor gettext have direct, or obvious indirect, dependencies on these three packages.

Reply:
Sure!